test/Table_Tests/src/Common_Table_Operations/Column_Operations_Spec.enso

```diff
@@ -925,6 +925,70 @@ spec setup =
             t = table_builder [["numbers", [1, 2, 3]]]
             col = t.at "numbers"
             col.text_length . should_fail_with Invalid_Value_Type
+        
+        Test.specify "should handle operation text_left and text_right with length 1" <|
+            with_mixed_columns_if_supported [["strings", ["a", "foobar", "", Nothing, "café", "It was the best of times, it was the worst of times, it was the age of wisdom, it was the age of foolishness, it was the epoch of belief, it was the epoch of incredulity, it was the season of light, it was the season of darkness, it was the spring of hope, it was the winter of despair."]]] t->
+                col = t.at "strings" . cast (Value_Type.Char size=286 variable_length=True)
+                resLeft = col.text_left 1
+                resLeft.name . should_equal "text_left([strings], 1)"
+                resRight = col.text_right 1
+                resRight.name . should_equal "text_right([strings], 1)"
+                resLeft . to_vector . should_equal ["a", "f", "", Nothing, "c", "I"] 
+                resRight . to_vector . should_equal ["a", "r", "", Nothing, "é", "."]
+                case setup.is_database of
+                    False -> resLeft . value_type . should_equal (Value_Type.Char size=286 variable_length=True)
+                        resRight . value_type . should_equal (Value_Type.Char size=286 variable_length=True)
+                    True -> resLeft . value_type . should_equal (Value_Type.Char variable_length=True)
+                        resRight . value_type . should_equal (Value_Type.Char variable_length=True)
```

Comment:
Btw. are you sure this runs?

A similar example
```
from Standard.Base import all

main =
    x = True
    case x of
        True -> IO.println "A"
            IO.println "B"
        False -> IO.println "C"
            IO.println "D"
```
fails for me with
```
A
Execution finished with an error: Type error: expected a function, but got Nothing.
        at <enso> case_branch(indents.enso:6:17-30)
        at <enso> indents.main(indents.enso:5-9)
```

```suggestion
                    False -> 
                        resLeft . value_type . should_equal (Value_Type.Char size=286 variable_length=True)
                        resRight . value_type . should_equal (Value_Type.Char size=286 variable_length=True)
                    True -> 
                        resLeft . value_type . should_equal (Value_Type.Char variable_length=True)
                        resRight . value_type . should_equal (Value_Type.Char variable_length=True)
```

Reply:
Hmm, the tests seem to be green - I'm wondering how is that possible 😅

Reply:
Yes they do run somehow. If the first test in the case fails though you don't get a good failure message.

I will fix to something more conventional